server/app.js

```diff
@@ -49,13 +49,25 @@ server.listen(port, hostName, function () {
     console.log(`Server1 running on http://${hostName}:${port}`);
 });
 
-//create and return a new board
-app.get(
-    "/api/newboard",
-
-    (req, res) => {
-        res.json(newBoard());
+//create and return a new board based on user's dictionary
+app.get("/api/newBoard", (req, res) => {
```

Comment:
This might have to be app.post instead of app.get, that way the req will actually have a body. Something to test out.
@GonzaloAllenPerez444 maybe verify this with postman?

Reply:
You should definitely make this a post request. The form data will be sent over as JSON, so you can use the "body-parser" middleware to parse it before you put it into a variable. It should be pretty easy, I'd recommend asking chat GPT how to implement it.